.eslintrc.js

```diff
@@ -100,6 +100,11 @@ module.exports = {
                 "@typescript-eslint/semi": 2,
                 "@typescript-eslint/no-empty-function": 0,
                 "@typescript-eslint/no-use-before-define": 0,
+                // We have to use any on some types so the warning isn't valuable.
```

Comment:
:(

Reply:
Yeah :( Long term we'll use the TS compiler strictness options to get rid of this but these warnings are adding no value.